freqtrade/exchange/exchange.py

```diff
@@ -187,6 +187,11 @@ def __init__(self, config: Config, *, exchange_config: Optional[ExchangeConfig]
         self._api_async = self._init_ccxt(
             exchange_conf, ccxt_async, ccxt_kwargs=ccxt_async_config)
 
+        if exchange_conf.get('sandboxMode'):
+            self._api.set_sandbox_mode(True)
+            self._api_async.set_sandbox_mode(True)
```

Comment:
it seems that the delist endpoint is not available on the the testnet 

but not sure if we could keep this since there are quite some cex's out there that allow papertrading

Reply:
why don't you use the actual server instead of the testnet? FT has been discouraging the use of testnet because of the limited features in it.

Reply:
the problem that I'm having is that the delist endpoint is actually an authenticated endpoint 

In dry_mode this will fail since we are not using the authentication there

Reply:
it's not the first (and won't be the last) feature that ain't fully supported in dry-run mode. 

it's also really a topic of "it's an authenticated endpoint on binance" - if you look at the same / corresponding endpoint on bybit - then it's not authenticated.